### DISCUSSION TITLE
Mail Filtering

### DISCUSSION THREAD

**Discussion by Pairs34** [anonaddy/anonaddy#149] [Feature Requests]
Hi , can u add mail filtering.

**Comment by willbrowningme** [comment]
Could you please provide more details as to what you mean by this?

**Comment by Pairs34** [comment]
I want to blocking some emails for forwarding. Because a lot of emails
exceeded my forwarding.

4 May 2021 Sal 13:36 tarihinde Will Browning ***@***.***>
şunu yazdı:

**Reply by willbrowningme** [reply]
At the moment the only way to block emails from being forwarded is to deactivate or delete the alias in question.

I am working on a rules system that will allow more granular control over which senders are blocked etc.
